package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lev-react-components",
-  "version": "0.8.0",
+  "version": "0.8.1",
```

Comment:
This should be v0.9.0 as it contains brand new components.

But it's not a big deal.

Reply:
Done

Reply:
Did you forget to push?